packages/react/src/provider.tsx

```diff
@@ -2920,6 +2922,10 @@ export const PhotonProvider = (opts: {
     client.setOrganization(organizationId);
   };
 
+  const removeOrganization = () => {
+    client.removeOrganization();
+  };
+
```

Comment:
not sure if i like this, was also thinking setOrganization can take an optional parameter and leaving it out clears the organization